pantheon-bundle/src/main/resources/SLING-INF/content/docs/attributes.adoc

```diff
@@ -13,3 +13,4 @@
 :PublishingPortal: Customer Portal
 
 :imagesdir: images
+:content: content
```

Comment:
Hey @stoobie - during our previous sprint meeting, we settled on :ContentTerm: for this one. Please update references to use this attribute, otherwise, everything looks great to me!